### PR TITLE
Fix modal panels

### PR DESCRIPTION
**Pull request by NateWr**

### CODE REVIEW

File: js/controllers/modal/ModalHandler.js [hunk 1]
```diff
@@ -47,7 +47,7 @@
 				titleIcon: string }} */ (this.mergeOptions(internalOptions));
 
 		// Attach content to the modal
-		$handledElement.html(this.modalBuild().html());
+		$handledElement.html(this.modalBuild()[0].outerHTML);
```
Comment (asmecher) [inline js/controllers/modal/ModalHandler.js]: Sorry, my bug -- this was in response to a JS linting error but I didn't test the change.

Comment (NateWr) [inline js/controllers/modal/ModalHandler.js]: No worries. Just panicked me for a bit. :)

Comment (asmecher) [inline js/controllers/modal/ModalHandler.js]: You probably saw this, but I kept my tweaks as separate commits from the squashed one containing your work. That'll hopefully make it easier to track down anything I inadvertently introduced.

Comment (NateWr) [inline js/controllers/modal/ModalHandler.js]: That would have been the smart thing to check. No, I went and just generated a big diff of js files. :)

**Pull request merged**